<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ourchatdialog</class>
 <widget class="QWidget" name="Ourchatdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>567</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>567</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>191</width>
     <height>321</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>189</width>
      <height>319</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="our_wdg" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>282</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_commit">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>320</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
